README.md

```diff
@@ -68,23 +71,6 @@ Or, just to compile them:
 make tests
 ```
 
-## Support
-
-Should you need assistance with New Relic products, you are in good hands with several support channels.
-
-If the issue has been confirmed as a bug or is a feature request, file a GitHub issue.
-
-**Support Channels**
-
-* [New Relic Documentation](https://docs.newrelic.com/docs/c-sdk-table-contents): Comprehensive guidance for using our platform
-* [New Relic C SDK documentation on GitHub](https://newrelic.github.io/c-sdk/index.html)
-* [New Relic C SDK Guide](https://github.com/newrelic/c-sdk/blob/master/GUIDE.md)
-* [New Relic C SDK distributed tracing example](https://github.com/newrelic/c-dt-example)
-* [New Relic Community](https://discuss.newrelic.com/c/build-on-new-relic/Open-Source-Agents-SDKs): The best place to engage in troubleshooting questions
-* [New Relic Developer](https://developer.newrelic.com/): Resources for building a custom observability applications
-* [New Relic University](https://learn.newrelic.com/): A range of online training for New Relic users of every level
-* Use your preferred search engine to find other New Relic resources.
-
```

Comment:
Is New Relic going to remove all C SDK documentation from their website? If no, then I'd suggest to leave these links in and only remove the link to `New Relic Community`.